src/common/estimate_size/src/collections/vec.rs

```diff
@@ -15,12 +15,12 @@
 use crate::EstimateSize;
 
 #[derive(Clone)]
-pub struct VecWithKvSize<T: EstimateSize> {
+pub struct EstimatedVec<T: EstimateSize> {
     inner: Vec<T>,
     kv_heap_size: usize,
```

Comment:
This is counterintuitive. 😄 What does "heap" and "kv" mean in the context of a vector?

Reply:
fixed